src/Features/CSharp/Portable/Microsoft.CodeAnalysis.CSharp.Features.csproj

```diff
@@ -28,6 +28,7 @@
     <!-- BEGIN External Access -->
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.ExternalAccess.OmniSharp.CSharp" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.ExternalAccess.OmniSharp.UnitTests" />
+    <InternalsVisibleTo Include="Microsoft.CodeAnalysis.ExternalAccess.AspNetCore" />
```

Comment:
I believe all the IVTs for EA layer should be `RestrictedInternalsVisibleTo` not `InternalsVisibleTo` - this enables the [RestrictedInternalsVisibleToAnalyzer](https://github.com/dotnet/roslyn-analyzers/blob/main/src/Microsoft.CodeAnalysis.BannedApiAnalyzers/Core/RestrictedInternalsVisibleToAnalyzer.cs) on the accessing project to ensure that it only accesses APIs within the restricted namespaces that it has been given access to, not the entire internal surface area.

Reply:
Seems like this was done wrong for both `OmniSharp` (above) and `Razor` (below)

Reply:
done!